src/components/singleArticle.jsx

```diff
@@ -17,7 +18,34 @@ const SingleArticle = () => {
         })
     })
 
+    const increaseVoteCount = () => {
+        console.log(currentArticle)
+        setCurrentArticle({
```

Comment:
You're right that there is an issue with your optimistic rendering. The way you are updating your current article isn't quite there. We're only targeting the vote count, nothing else. You already have access to this and would be wise to hold in state.